test/e2e/tests/settings-add-snap-account-toggle.spec.js

```diff
@@ -0,0 +1,52 @@
+const { strict: assert } = require('assert');
+const { withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Add snap account experimental settings', function () {
+  it('switch "Enable Add snap account" to on', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // Make sure the "Add snap account" button is not visible.
+        await driver.clickElement('[data-testid="account-menu-icon"]');
+        await driver.assertElementNotPresent({
+          text: 'Add snap account',
+          tag: 'button',
+        });
+        await driver.clickElement('.mm-box button[aria-label="Close"]');
+
```

Comment:
For the ease of troubleshooting potential test failures going forward, will recommend that we consider separating this single block into two.

it ('add snap account disabled by default`...
and 
it ('add snap account after experimental setting enabled'....